src/index.html

```diff
@@ -3,11 +3,320 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link rel="icon" href="./images/logo.png">
+    <title>Nothing</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Landing Page</h1>
+    <header class="header">
+      <div class="header__top top-bar">
+        <a href="#">
+          <img
+            class="top-bar__logo"
+            src="images/logo.png"
+            alt="nothing logo"
+          >
+        </a>
+
+        <div class="top-bar__icons">
+          <a href="#" class="icon icon--phone"></a>
+
```

Comment:
```suggestion
```

Reply:
It's two tags on one line, there is an empty line must be between them.